core/src/test/java/org/apache/calcite/test/JdbcAdapterTest.java

```diff
@@ -542,6 +550,31 @@
             + "FROM \"foodmart\".\"expense_fact\"");
   }
 
+  @Test void testTableScanHint() {
+    SqlNodeList hints = HintSqlDialectFactory.captureHints(() -> CalciteAssert
+        .model(JdbcTest.FOODMART_MODEL_WITH_HINTS)
+        .enable(CalciteAssert.DB == DatabaseInstance.HSQLDB)
+        .query("?")
+        .withRel(
+            builder -> {
+              builder.getCluster().setHintStrategies(HintStrategyTable.builder()
+                  .hintStrategy("PLACEHOLDERS", HintPredicates.TABLE_SCAN)
+                  .build());
+              return builder
+                  .scan("foodmart", "expense_fact")
+                  .hints(RelHint.builder("PLACEHOLDERS")
+                      .hintOptions(ImmutableMap.of("a", "b"))
```

Comment:
You can use this method instead `hintOption(String optionKey, String optionValue)`.

Reply:
I will change it accordingly.